### ISSUE TITLE
Add monero and bitcoin donation addresses

### ISSUE THREAD

**Issue by peepo5** [tenacityteam/tenacity-legacy#655]
### Guidelines

- [X] I have read the guidelines.

### Is your feature request related to a problem? Please describe.

I am frustrated when I cannot donate using bitcoin or monero

### Describe the solution you'd like

To add bitcoin and monero donation addresses

### Describe alternatives you've considered

Not donating

### This issue is not a duplicate

- [X] I have confirmed this issue isn't a duplicate.

**Comment by Semisol** [comment]
OpenCollective now supports crypto, but it is enabled on demand. Also, it is converted to fiat after being received.

**Comment by peepo5** [comment]
That seems against the point.
If you don't own the wallet you don't own the funds.

**Comment by Semisol** [comment]
A few problems:
- Who will hold the keys? How do we prevent any of us from stealing the key and running away with the funds?
- Transparency is harder with Bitcoin & Monero, as there is no way to easily attach a "note"
- Most of our expenses will probably be in dollars, so it will need to be converted either way.

**Comment by peepo5** [comment]
if opencollective does not support monero then you an address could be added, however bitcoin is not anonymous anyways so i guess opencollective could be used.

**Comment by Semisol** [comment]
> if opencollective does not support monero then you an address could be added, however bitcoin is not anonymous anyways so i guess opencollective could be used.

Monero might not be supported, but Bitcoin should be. I'll look into it later.
I don't know if it is currently worth it to ask for crypto support on the collective.

**Comment by n0toose** [comment]
>  If you don't own the wallet you don't own the funds.

Well, that's kind of the goal here. As with the rest of the monies that have been donated so far (approximately 350-400 USD, last time I checked), this money goes to our fiscal host, Open Collective, that manages the money and legal paperwork. Although we've had our fair share of issues, this sort of level of security adds an additional level of trustworthiness to the people that are donating that we won't, say, hypothetically abuse that money, and it also reassures other contributors that we're not going to do that. We also wouldn't want a situation similar to the one of another Audacity fork, which ended up losing the private keys to their funds. We did discuss all of that earlier on in private and put off the entire topic of crypto for the time being for those reasons (as well as others I may not recall).

> Bitcoin and Monero

Although I am interested in blockchain technologies in general, I don't personally find it moral to accept and further "validate" PoW coins in the presence of better alternatives that are still somewhat flawed, but at the very least somewhat friendlier to the environment, while the climate crisis is a thing. Not sure about everyone else.
